tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -0,0 +1,138 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-categorization-utils/tf-categorization-utils.html">
+<link rel="import" href="../tf-categorization-utils/tf-category-pane.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-dashboard-common/dashboard-style.html">
+<link rel="import" href="../tf-dashboard-common/tf-dashboard-layout.html">
+<link rel="import" href="../tf-runs-selector/tf-runs-selector.html">
+<link rel="import" href="tf-pr-curve-card.html">
+<link rel="import" href="tf-step-selector.html">
+
+<!--
+  A frontend that displays a set of precision-recall curves. Each PR curve is
+  associated with a tag (which is in turn associated with 1 class).
+-->
+<dom-module id="tf-pr-curve-dashboard">
+  <template>
+    <tf-dashboard-layout>
+      <div class="sidebar">
+        <div class="sidebar-section">
+          <h3>Step for Each Run</h3>
+          <p>
+            Runs may differ in current step. For each run, we show the PR curve
+            at the largest step no greater than the slider step.
+          </p>
+          <template is="dom-repeat" items="{{_runStepPairs}}">
+            <div class="run-step-combo">
+              <div class="run-text"
+                    style="color:[[_computeColorForRun(item.run)]];">
+                [[item.run]]
+              </div>
+              <div class="step-text">[[item.step]]</div>
```

Comment:
There should be sliders here: each run needs to have its current step individually controlled. (Consult original design for more info: my email "State and next-steps for TensorBoard PR curves" from 2017-07-31.)